src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -490,7 +490,7 @@ public boolean quit() {
                 context.clearDBMSSynchronizer();
             }
             AutosaveManager.shutdown(context);
-            BackupManager.shutdown(context);
+            BackupManager.shutdown(context, prefs.getFilePreferences().getBackupDirectory(), prefs.getFilePreferences().shouldCreateBackup());
```

Comment:
thought: three arguments is a bit too much for a static method. I think it is time to convert BackupManager to use instance methods.

Reply:
The backup manager is started early for each bib file. That cannot be changed on the fly. Maybe add this to the restart warnings 

The BackupManager is a combination of static and instance methods. Basically, the BackupManager has a static set with all running instances.  Start adds a new instance of the backup manager. A backup manager is created for each bibdatabasecontext and the running instances set ensures that there is at most one for each bib file.